composer.json

```diff
@@ -10,14 +10,21 @@
     ],
     "minimum-stability" : "dev",
     "require": {
-        "php": ">=5.3",
-        "fliglio/web": "dev-master",
-        "guzzlehttp/guzzle": "~5.0"
+        "php"               : ">=5.6",
+        "fliglio/web"       : "dev-master",
+        "guzzlehttp/guzzle" : "~5.0"
```

Comment:
is the guzzle usage under test & compatible with 5.6 & 7?

Reply:
There was some coverage and I added more. Guzzle 5+ is compatible with php55+. Doesn't look like it limits it to only 5. Tests show it works in 7.